--- origsrc/libilbc/CMakeLists.txt	2013-01-27 20:39:58.000000000 -0600
+++ src/libilbc/CMakeLists.txt	2013-01-27 20:48:45.481818600 -0600
@@ -26,7 +26,7 @@ set_target_properties(ilbc PROPERTIES CL
 set_target_properties(ilbc-static PROPERTIES OUTPUT_NAME "ilbc")
 set_target_properties(ilbc-static PROPERTIES CLEAN_DIRECT_OUTPUT 1)
 
-set(CMAKE_C_FLAGS "-O2 -Wall -Wextra -Werror -fvisibility=hidden -Wno-unused-parameter -fno-strict-aliasing")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2 -Wall -Wextra -Werror -Wno-unused-parameter -fno-strict-aliasing")
 
 set(prefix "${CMAKE_INSTALL_PREFIX}")
 set(exec_prefix "\${prefix}")
